### PR TITLE
Invariant Improvements: Add invariant scenario for multiple fund treasury between distributions

### PR DESCRIPTION
**Pull request by prat-gpt**
# Description of change
## High level
* Add invariant scenario with multiple treasury funding, transfer ajna between actors and new actors being added.

### CODE REVIEW

File: test/invariants/handlers/Handler.sol [hunk 1]
```diff
@@ -70,7 +70,10 @@ contract Handler is Test, GrantFundTestHelper {
         _tokenDeployer = tokenDeployer_;
 
         // instantiate actors
-        actors = _buildActors(numOfActors_, tokensToDistribute_);
+        address[] memory newActors = _buildActors(numOfActors_, tokensToDistribute_);
+        for (uint256 i = 0; i < newActors.length; ++i) {
+            if (newActors[i] != address(0)) actors.push(newActors[i]);
```
Comment (MikeHathaway) [inline test/invariants/handlers/Handler.sol:75]: Were any of the newActors addresses actually `0x0`?

**Review by MikeHathaway — APPROVED** [review]
LGTM

File: test/invariants/handlers/Handler.sol (hunk 1 above)
Comment (prat-gpt) [reply test/invariants/handlers/Handler.sol:75]: yes, i got some failures where actor addresses were `0x0`, it was happening [here](https://github.com/ajna-finance/ecosystem-coordination/blob/16f0cfe6e49622470333277815d91ae91c327e61/test/invariants/handlers/Handler.sol#L155).

**Review by EdNoepel — APPROVED** [review]
LGTM

File: test/invariants/handlers/Handler.sol (hunk 1 above)
Comment (MikeHathaway) [reply test/invariants/handlers/Handler.sol:75]: Ah, yeah from the failure path. Makes sense

**Pull request merged**